src/helix-ui/elements/HXModalElement.js

```diff
@@ -0,0 +1,77 @@
+import { HXElement } from './HXElement';
+import { KEYS } from '../util';
+import shadowStyles from './_hx-modal.less';
+
+const tagName = 'hx-modal';
+const template = document.createElement('template');
+
+template.innerHTML = `
+  <style>${shadowStyles}</style>
+  <div id="container">
+    <button id="close">
+      <hx-icon type="times"></hx-icon>
+    </button>
+    <slot></slot>
+  </div>
+`;
+
+export class HXModalElement extends HXElement {
+    static get is () {
+        return tagName;
+    }
+
+    static get observedAttributes () {
+        return [ 'open' ];
+    }
+
+    constructor () {
+        super(tagName, template);
+        this._toggle = this._toggle.bind(this);
+        this._keyUp = this._keyUp.bind(this);
+    }
+
+    connectedCallback () {
+        this.$upgradeProperty('open');
+        this._target = this.shadowRoot.querySelector("#close");
+
+        this._target.addEventListener('click', this._toggle);
+        document.addEventListener('keyup', this._keyUp);
+    }
+
+    disconnectedCallback () {
+        this._target.removeEventListener('click', this._toggle);
+        document.removeEventListener('keyup', this._keyUp);
+    }
+
+    attributeChangedCallback (attr, oldValue, newValue) {
+        this.setAttribute('aria-hidden', newValue !== '');
+    }
+
+    _toggle () {
```

Comment:
Given that this function is only being used to close the modal (when clicking the close button), how about renaming it to `_close()` and changing the logic to `this.open = false;`?

```javascript
_close () {
    this.open = false;
}
```